### PR TITLE
Disable flakey ustack tests in CI

### PR DESCRIPTION
**Pull request by danobi**
See commits for more details

##### Checklist

- [ ] Language changes are updated in `man/adoc/bpftrace.adoc`
- [ ] User-visible and non-trivial changes updated in `CHANGELOG.md`
- [ ] The new behaviour is covered by tests

### CODE REVIEW

File: tests/runtime/call [hunk 1]
```diff
@@ -232,27 +232,44 @@ ENV BPFTRACE_STACK_MODE=bpftrace
...
+# [1]: https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables
+SKIP_IF_ENV_HAS CI=true
 
 NAME ustack_stack_mode_env_perf
 PROG u:./testprogs/uprobe_loop:uprobeFunction1 { printf("%s\n", ustack(1)); exit(); }
 ENV BPFTRACE_STACK_MODE=perf
 EXPECT_REGEX \d+ uprobeFunction1\+0 \(.*/uprobe_loop\)
 TIMEOUT 5
 AFTER ./testprogs/uprobe_loop
+# See above
```
Comment (jordalgo) [inline tests/runtime/call:253]: Can we add a link to an existing issue here instead of `#See above` ?

File: tests/README.md [hunk 2]
```diff
@@ -97,6 +97,8 @@ Each runtime testcase consists of multiple directives. In no particular order:
 * `NEW_PIDNS`: This will execute the `BEFORE`, the bpftrace (`RUN` or `PROG`),
   and the `AFTER` commands in a new pid namespace that mounts proc. At least one
   `BEFORE` is required.
+* `SKIP_IF_ENV_HAS`: Skip test case if specified environment variable is found
```
Comment (jordalgo) [inline tests/README.md:100]: Maybe we should mention that only a single KEY/VALUE is per test is allowed

**Comment by jordalgo** [comment]
Few small nits but overall I like this approach better, that is the skipping directive being closer to the test themselves.

File: tests/runtime/call (hunk 1 above)
Comment (danobi) [reply tests/runtime/call:253]: Yeah good idea

**Review by jordalgo — APPROVED** [review]

**Pull request merged**